posthog/settings/web.py

```diff
@@ -216,14 +216,19 @@
 if DEBUG:
     REST_FRAMEWORK["DEFAULT_RENDERER_CLASSES"].append("rest_framework.renderers.BrowsableAPIRenderer")  # type: ignore
 
-if get_from_env("RATE_LIMIT_ENABLED", False, type_cast=str_to_bool) or TEST:
+RATE_LIMIT_ENABLED = get_from_env("RATE_LIMIT_ENABLED", True, type_cast=str_to_bool)
+
+if RATE_LIMIT_ENABLED or TEST:
     # These rate limits are applied to all Django views.
     # Note: Ingestion + decide endpoints do not use Django views, so no rate limits are applied
     REST_FRAMEWORK["DEFAULT_THROTTLE_CLASSES"] = [
         "posthog.rate_limit.PassThroughBurstRateThrottle",
         "posthog.rate_limit.PassThroughSustainedRateThrottle",
     ]
-    REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"] = {"burst": "120/minute", "sustained": "1000/hour"}
+    REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"] = {
```

Comment:
picking nits... changes to these defaults feels separate. Alters whatever was being tested with the pass through throttles without impacting this change?

Reply:
Good catch - not intentional at all